geos/entrypoints.go

```diff
@@ -231,6 +231,15 @@ func Simplify(g geom.Geometry, tolerance float64) (geom.Geometry, error) {
 	return rawgeos.Simplify(g, tolerance)
 }
 
+// TopologyPresereSimplify creates a simplified version of a geometry using
```

Comment:
Same with func name and use in test. 😄 

```suggestion
// TopologyPreserveSimplify creates a simplified version of a geometry using
```

Reply:
Oops! Thanks for noticing that! (cde8f02015bc32707820e0561fa4cdda6b9971b3).